feature/bgp/policybase/otg_tests/import_export_multi_test/import_export_multi_test.go

```
@@ -860,6 +894,60 @@ func validateOTGBgpPrefixV4AndASLocalPrefMED(t *testing.T, otg *otg.OTG, dut *on
 	}
 }
 
+// configureAsPolicy is used to configure vendor specific config statement.
+func configureAsPolicy(t *testing.T, dut *ondatra.DUTDevice) {
+	t.Helper()
+	var config string
+	gnmiClient := dut.RawAPIs().GNMI(t)
+
+	switch dut.Vendor() {
+	case ondatra.JUNIPER:
+		config = juniperCLI()
```

I don not agree on the use of CLI to pass a test, if a feature is not supported let it fail until supported.

---

Removed CLI config